prov/efa/src/rdm/rxr_ep.h

```diff
@@ -234,6 +234,8 @@ struct rxr_ep {
 	int blocking_copy_rx_entry_num; /* number of RX entries that are using gdrcopy/cudaMemcpy */
 
 	int	hmem_p2p_opt; /* what to do for hmem transfers */
+
+	struct fid_peer_srx peer_srx; /* support sharing receive context with peer providers */
```

Comment:
Suggest moving this to L74 because of the its significance and relevance to SHM

Reply:
Conceptually this `peer` can be any provider, not for shm specifically. I do not think making it close to `shm` variables make more sense.